web/basics/binding_to_a_complex_object/my_element.html

```diff
@@ -1,7 +1,7 @@
-<polymer-element name="my-element">
-  <template>
-    <div>{{person.name.first}} {{person.name.last}}</div>
-    <div>{{person.title}}</div>
-    <div>{{person.company.name}}</div>
-  </template>
-</polymer-element>
+<dom-module name="my-element">
```

Comment:
fwiw it looks like this does actually work (news to me!) but using `id` is still the recommended solution in the docs so I would stick with that.